tests/conftest.py

```diff
@@ -57,3 +57,49 @@ def pytest_addoption(parser: Parser):
         action="store",
         help="The OpenStack clouds yaml file for the charm to use.",
     )
+    # Private endpoint options
```

Comment:
I think these options are technically not a part of this PR right?
Since this PR is merging into main, it might be good to include this yet.

Reply:
Since the secrets are set on the repository, the test will fail with `pytest unrecognized args` unfortunately. [Link to example failed run](https://github.com/canonical/github-runner-operator/actions/runs/9001993048/job/24729552447#step:22:17).